doc/api/next_api_changes/deprecations/22813-GL.rst

```diff
@@ -0,0 +1,5 @@
+``figure.callbacks`` is deprecated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The figure callbacks property is deprecated. The only signal was
+"dpi_changed", which can be replaced by connecting to the "resize_event"
+on ``figure.canvas.callbacks`` instead.
```

Comment:
Should this point to `canvas.mpl_connect` instead?

Reply:
Sure, I updated the doc. `canvas.mpl_connect` is simply an alias for `canvas.callbacks.connect` it looks like. I guess `mpl_connect` is preferred?